lib/terrafying/components/letsencrypt.rb

```diff
@@ -135,7 +137,7 @@ def create_keypair_in(ctx, name, options={})
         ctx.resource :aws_s3_bucket_object, "#{key_ident}-cert", {
                        bucket: @bucket,
                        key: File.join(@prefix, @name, name, "cert"),
-                       content: output_of(:acme_certificate, key_ident, :certificate_pem),
+                       content: output_of(:acme_certificate, key_ident, :certificate_pem) + @ca_cert,
```

Comment:
This will result in a terraform interpolated string `${acme_certificate.key_ident.certificate_pem}` being concatted with a cert pem.

For this to work we'll need to somehow append pem content to a cert we don't have yet, so might need some more investigation first.

Reply:
That should be fine, as long as there's a new line